Source/Core/Core/PowerPC/Interpreter/Interpreter.cpp

```diff
@@ -147,10 +117,12 @@ bool Interpreter::HandleFunctionHooking(u32 address)
 
 int Interpreter::SingleStepInner()
 {
+  const GekkoOPInfo* opinfo = PPCTables::GetOpInfo(m_prev_inst);
+
   if (HandleFunctionHooking(PowerPC::ppcState.pc))
   {
     UpdatePC();
-    return PPCTables::GetOpInfo(m_prev_inst)->numCycles;
+    return opinfo->num_cycles;
   }
```

Comment:
Hmm, does that mean that this code is incorrect, as it meant that a hooked function uses the cycle count for the previous instruction instead of the current instruction? Not that exact cycle counts mean anything if we're hooking a function.

Reply:
I think you're right, yeah, at least I don't see any place `m_prev_inst` would be updated in this case to point at the current instruction. Probably better fixed in another PR though.

Reply:
Regression fixed. I've also added a TODO for this code.